README.md
# qPod

A [GNUpod](http://www.gnu.org/software/gnupod/) frontend written with QT 
It was published at 2007

Web site : [http://qpod.sourceforge.net](http://qpod.sourceforge.net)




src/Changelog
Beta1-
-----
* Uygulama hazirlandi
->Bilinen hatalar
-- gnupod_INIT calismiyor
-- sarki sırası değiştirilemiyor


Beta2
----
* gnupod_INIT sorunu çözüldü
-> Bİlinen hatalar
-- Sarki sırası değiştirilemiyor 


src/qpod.h
#ifndef QPOD_H
#define QPOD_H
#include <QProcess>
#include <QAction>
#include <QIcon>
#include <QTranslator>
#include <QSettings>

#include "ui_anaPencere.h"
 
class qPodSecenekMenusu;

class qPod : public QMainWindow, private Ui::anaPencere
{
    Q_OBJECT
 
public:
qPod(QMainWindow *parent = 0);
QTranslator translator;
void boyutDegistir(int x, int y);
void ayarlariOku();
void dilDegistir();
void guvenliKaldir( QCloseEvent *event);
QString aygit,baglamaNoktasi;
QString gnupod_tunes2pod, gnupod_addsong, gnupod_search, gnupod_check, gnupod_mktunes; //komutlar
QProcess islem;
QSettings *secenekler;
bool baglandi,cikarildi;
QWidget *ayarlarTab, *ipodTab,*yuklenecekSarkilarTab;
int seciliRow;


public slots:
bool gnupodYollariniKontrolEt();
void baglan();
void ayarlariKapat();
void ayarlariKaydet();
void ayarlarOntanimli();
void ayarlarYardim();
void ipodSarkiListesiAl();
void silButtonEtkin();
void sarkilariSil();
void sarkilariYukle();
void ayarlar();
void sarkiSec();
void siraDegistir();
void siralamayiYaz();


protected:
 void dragEnterEvent(QDragEnterEvent *event);
     void dragMoveEvent(QDragMoveEvent *event);
     void dropEvent(QDropEvent *event);
     void closeEvent ( QCloseEvent * event );
};


 
#endif 


src/ceviri/qpod_tr.ts
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS><TS version="1.1">
<context>
    <name>anaPencere</name>
    <message>
        <location filename="../anaPencere.ui" line="13"/>
        <source>qPod</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="364"/>
        <source>background-color:white;</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="95"/>
        <source>background:none;</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../anaPencere.ui" line="60"/>
        <source>Bağlan</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="352"/>
        <source>Ayarlar</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../anaPencere.ui" line="131"/>
        <source>Yüklenecek Şarkılar</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="146"/>
        <source>font-size:10px;</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../anaPencere.ui" line="187"/>
        <source>Şarkı Seç</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="206"/>
        <source>iPod</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="255"/>
        <source>font-size:9px;</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../anaPencere.ui" line="292"/>
        <source>Seçili Şarkıları Sil</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../anaPencere.ui" line="371"/>
        <source>Aygıtlar</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../anaPencere.ui" line="383"/>
        <source>Aygıt</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../anaPencere.ui" line="399"/>
        <source>Bağlama Noktası</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="638"/>
        <source>Uygula</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../anaPencere.ui" line="479"/>
        <source>Öntanımlı</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="454"/>
        <source>gnupod</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="515"/>
        <source>gnupod_addsong</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="528"/>
        <source>gnupod_search</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="541"/>
        <source>gnupod_check</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="554"/>
        <source>mktunes</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="600"/>
        <source>Dil</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="613"/>
        <source>English</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../anaPencere.ui" line="621"/>
        <source>Türkçe</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../anaPencere.ui" line="653"/>
        <source>Ayarları Kapat</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../anaPencere.ui" line="174"/>
        <source>Şarkıları iPod&apos;a Yükle</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="502"/>
        <source>tunes2pod</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>qPod</name>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="76"/>
        <source>Bir aygıt belirtilmemiş</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="77"/>
        <source>Bir bağlama noktası belirtilmemiş</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="89"/>
        <source>Bağlantı sırasında bir hata oluştu, lütfen ayarlarınız kontrol edin</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="731"/>
        <source>Evet</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="732"/>
        <source>Hayır</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="706"/>
        <source>HATA</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="106"/>
        <source>iPod&apos;unuzu daha önce qPod ile kullanmamış görünyorsunuz.&lt;br/&gt;Sizin için gerekli dosyaları yüklememi ister misiniz?</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="163"/>
        <source>Bağlantıyı kes</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="165"/>
        <source>iPod</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="166"/>
        <source>Yüklenecek Şarkılar</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="338"/>
        <source>Tamam</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="211"/>
        <source>Bağlan</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="232"/>
        <source>Bağlantının kesilmesi sırasında bir hata oluştu&lt;br/&gt;&lt;b&gt;Hata:&lt;/b&gt;&lt;br/&gt;</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="246"/>
        <source>gnupod_search yolu eksik</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="247"/>
        <source>gnupod_addsong yolu eksik</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="248"/>
        <source>gnupod_check yolu eksik</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="249"/>
        <source>mktunes yolu eksik</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="255"/>
        <source>gnupod_search yolu yanlış</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="256"/>
        <source>gnupod_addsong yolu yanlış</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="257"/>
        <source>gnupod_check yolu yanlış</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="258"/>
        <source>mktunes yolu yanlış</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="287"/>
        <source>Lütfen ayarlar menüsünden aşağıdaki hataları düzeltin.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="665"/>
        <source>Ayarlar</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="427"/>
        <source>Şarkı</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="428"/>
        <source>Şarkıcı</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="429"/>
        <source>Albüm</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="430"/>
        <source>#</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="527"/>
        <source>Şarkı Silmek</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="344"/>
        <source>Yaptığınız dil değişikliği uygulama yeniden başlatılınca etkin olacaktır</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="528"/>
        <source>Seçili şarkıları iPod&apos;unuzdan silmek istediğinize emin misiniz?</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="686"/>
        <source>Mp3 dosyaları seçin</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="733"/>
        <source>İptal</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="707"/>
        <source>Bağlantıyı kesmeyi unuttunuz!&lt;br/&gt;Sizin yerinize bağlantıyı kesmemi ister misiniz?</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="735"/>
        <source>Güvenli Kaldır</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="736"/>
        <source>iPod&apos;unuzu güvenli kaldırmak ister misiniz?</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="245"/>
        <source>tunes2pod yolu eksik</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="254"/>
        <source>tunes2pod yolu yanlış</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="809"/>
        <source>Hata</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="810"/>
        <source>Şarkı listesi değiştirilemiyor, bir hata oluştu!</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>


src/ceviri/qpod_en.ts
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS><TS version="1.1" language="en">
<defaultcodec></defaultcodec>
<context>
    <name>anaPencere</name>
    <message>
        <location filename="../anaPencere.ui" line="13"/>
        <source>qPod</source>
        <translation>qPod</translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="364"/>
        <source>background-color:white;</source>
        <translation></translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="95"/>
        <source>background:none;</source>
        <translation></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../anaPencere.ui" line="60"/>
        <source>Bağlan</source>
        <translation>Connect</translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="352"/>
        <source>Ayarlar</source>
        <translation>Settings</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../anaPencere.ui" line="131"/>
        <source>Yüklenecek Şarkılar</source>
        <translation>Upload Songs</translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="146"/>
        <source>font-size:10px;</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../anaPencere.ui" line="187"/>
        <source>Şarkı Seç</source>
        <translation>Select Songs</translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="206"/>
        <source>iPod</source>
        <translation>iPod</translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="255"/>
        <source>font-size:9px;</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../anaPencere.ui" line="292"/>
        <source>Seçili Şarkıları Sil</source>
        <translation>Delete selected songs</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../anaPencere.ui" line="371"/>
        <source>Aygıtlar</source>
        <translation>Devices</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../anaPencere.ui" line="383"/>
        <source>Aygıt</source>
        <translation>Device</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../anaPencere.ui" line="399"/>
        <source>Bağlama Noktası</source>
        <translation>Mount Point</translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="638"/>
        <source>Uygula</source>
        <translation>Apply</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../anaPencere.ui" line="479"/>
        <source>Öntanımlı</source>
        <translation>Default</translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="454"/>
        <source>gnupod</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="515"/>
        <source>gnupod_addsong</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="528"/>
        <source>gnupod_search</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="541"/>
        <source>gnupod_check</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="554"/>
        <source>mktunes</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="600"/>
        <source>Dil</source>
        <translation>Language</translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="613"/>
        <source>English</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../anaPencere.ui" line="621"/>
        <source>Türkçe</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../anaPencere.ui" line="653"/>
        <source>Ayarları Kapat</source>
        <translation>Close Settings</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../anaPencere.ui" line="174"/>
        <source>Şarkıları iPod&apos;a Yükle</source>
        <translation>Upload songs to iPod</translation>
    </message>
    <message>
        <location filename="../anaPencere.ui" line="502"/>
        <source>tunes2pod</source>
        <translation type="unfinished">tunes2pod</translation>
    </message>
</context>
<context>
    <name>qPod</name>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="76"/>
        <source>Bir aygıt belirtilmemiş</source>
        <translation>Device not defined</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="77"/>
        <source>Bir bağlama noktası belirtilmemiş</source>
        <translation>Mount point not defined</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="89"/>
        <source>Bağlantı sırasında bir hata oluştu, lütfen ayarlarınız kontrol edin</source>
        <translation>An error occured while connecting to device. Please check your settings</translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="731"/>
        <source>Evet</source>
        <translation>Yes</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="732"/>
        <source>Hayır</source>
        <translation>No</translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="706"/>
        <source>HATA</source>
        <translation>ERROR</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="106"/>
        <source>iPod&apos;unuzu daha önce qPod ile kullanmamış görünyorsunuz.&lt;br/&gt;Sizin için gerekli dosyaları yüklememi ister misiniz?</source>
        <translation>It seems that this is the first time that you use qPod.&lt;br/&gt;Do you want me to set your iPod for qPod?</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="163"/>
        <source>Bağlantıyı kes</source>
        <translation>Disconnect</translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="165"/>
        <source>iPod</source>
        <translation>iPod</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="166"/>
        <source>Yüklenecek Şarkılar</source>
        <translation>Upload Songs</translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="338"/>
        <source>Tamam</source>
        <translation></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="211"/>
        <source>Bağlan</source>
        <translation>Connect</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="232"/>
        <source>Bağlantının kesilmesi sırasında bir hata oluştu&lt;br/&gt;&lt;b&gt;Hata:&lt;/b&gt;&lt;br/&gt;</source>
        <translation>Error occured while disconnecting&lt;br/&gt;&lt;b&gt;Error:&lt;/b&gt;&lt;br/&gt;</translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="216"/>
        <source>gnupod_INIT yolu eksik</source>
        <translation type="obsolete">Path for gnupod_INIT is missing</translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="246"/>
        <source>gnupod_search yolu eksik</source>
        <translation>Path for gnupod_search is missing</translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="247"/>
        <source>gnupod_addsong yolu eksik</source>
        <translation>Path for gnupod_addsong is missing</translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="248"/>
        <source>gnupod_check yolu eksik</source>
        <translation>Path for gnupod_check is missing</translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="249"/>
        <source>mktunes yolu eksik</source>
        <translation>Path for mktunes is missing</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="225"/>
        <source>gnupod_INIT yolu yanlış</source>
        <translation type="obsolete">Path for gnupod_INIT is wrong</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="255"/>
        <source>gnupod_search yolu yanlış</source>
        <translation>Path for gnupod_search is wrong</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="256"/>
        <source>gnupod_addsong yolu yanlış</source>
        <translation>Path for gnupod_addsong is wrong</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="257"/>
        <source>gnupod_check yolu yanlış</source>
        <translation>Path for gnupod_check is wrong</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="258"/>
        <source>mktunes yolu yanlış</source>
        <translation>Path for mktunes is wrong</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="287"/>
        <source>Lütfen ayarlar menüsünden aşağıdaki hataları düzeltin.</source>
        <translation>Please fix theese errors from Settings menu.</translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="665"/>
        <source>Ayarlar</source>
        <translation>Settings</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="427"/>
        <source>Şarkı</source>
        <translation>Song</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="428"/>
        <source>Şarkıcı</source>
        <translation>Artist</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="429"/>
        <source>Albüm</source>
        <translation>Album</translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="430"/>
        <source>#</source>
        <translation type="unfinished"></translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="527"/>
        <source>Şarkı Silmek</source>
        <translation>Deleting Songs</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="476"/>
        <source>Seçili şarkıları iPod&apos;unuzdan silmek istediğinize emin misiniz.</source>
        <translation type="obsolete">Do you want to delete those selected songs from your iPod.</translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="623"/>
        <source>Uygula</source>
        <translation type="obsolete">Apply</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="627"/>
        <source>Şarkıları iPod&apos;a Yükle</source>
        <translation type="obsolete">Upload songs to iPod</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="628"/>
        <source>Seçili Şarkıları Sil</source>
        <translation type="obsolete">Delete selected songs</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="629"/>
        <source>Ayarları Kapat</source>
        <translation type="obsolete">Close Settings</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="344"/>
        <source>Yaptığınız dil değişikliği uygulama yeniden başlatılınca etkin olacaktır</source>
        <translation>Language will be changed after application restarts</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="528"/>
        <source>Seçili şarkıları iPod&apos;unuzdan silmek istediğinize emin misiniz?</source>
        <translation>Do you want to delete those selected songs from your iPod?</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="686"/>
        <source>Mp3 dosyaları seçin</source>
        <translation>Select Mp3 files</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="733"/>
        <source>İptal</source>
        <translation>Cancel</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="707"/>
        <source>Bağlantıyı kesmeyi unuttunuz!&lt;br/&gt;Sizin yerinize bağlantıyı kesmemi ister misiniz?</source>
        <translation>You forget to disconnect your iPod!&lt;br/&gt;Do you want me to do it for you?</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="735"/>
        <source>Güvenli Kaldır</source>
        <translation>Eject</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="736"/>
        <source>iPod&apos;unuzu güvenli kaldırmak ister misiniz?</source>
        <translation>Do you want to eject your iPod?</translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="245"/>
        <source>tunes2pod yolu eksik</source>
        <translation type="unfinished">Path for tunes2pod is missing</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="254"/>
        <source>tunes2pod yolu yanlış</source>
        <translation type="unfinished">Path for tunes2pod is wrong</translation>
    </message>
    <message>
        <location filename="../qpod.cpp" line="809"/>
        <source>Hata</source>
        <translation type="unfinished">Error</translation>
    </message>
    <message encoding="UTF-8">
        <location filename="../qpod.cpp" line="810"/>
        <source>Şarkı listesi değiştirilemiyor, bir hata oluştu!</source>
        <translation type="unfinished">Cannot change the order of songs. An error occured!</translation>
    </message>
</context>
</TS>
